devicemodel/core/main.c

```diff
@@ -137,7 +138,7 @@ usage(int code)
 		"       %*s [--part_info part_info_name] [--enable_trusty] [--intr_monitor param_setting]\n"
 		"       %*s [--vtpm2 sock_path] [--virtio_poll interval] [--mac_seed seed_string]\n"
 		"       %*s [--vmcfg sub_options] [--dump vm_idx] [--ptdev_no_reset] [--debugexit] \n"
-		"       %*s [--logger-setting param_setting] <vm>\n"
+		"       %*s [--logger-setting param_setting] [--guest_notify_method method] <vm>\n"
```

Comment:
Is this correct? (you use `--pm_notify_channel` in the rest of this patchset)?

Reply:
Yes.

Reply:
OK, I'm not understanding this then. How do you use this `--guest_notify_method` parameter?

Reply:
Oh. Just get your point. This guest_notify_method is incorrect. It should be "--pm_notify_channel".
Thanks a lot for pointing this out.

Reply:
Will update the patch after fix failure reported by CI.